docs(fix): add steps to setup slack (#5046)
RasaHQ/rasa: docs/user-guide/connectors/slack.rst

  1. To create the app go to: https://api.slack.com/apps and click
     on *"Create New App"*.

>   2. Activate the following features: 

Reviewer: I think you need another line after this one in order to get the list to render properly

>     1. interactive components

Reviewer: ```suggestion
    1. [interactive components](https://api.slack.com/reference/block-kit/interactive-components) (if your bot uses any, e.g. buttons)
```

    2. event subscriptions
    3. bot users

>     4. permissions (for basic functionality you should subscribe to the 

Reviewer: Please capitalize each element now that it is a list

       following events
       ``message.channel``,
       ``message.groups``, 
       ``message.im`` 

>        and ``message.mpim``)

Reviewer: following events: ``message.channel``,``message.groups``, ``message.im`` and ``message.mpim``)

.. code-block:: bash

  rasa run
  

> Do not forget to handle the actions if this is required by your bot, 

Reviewer: ```suggestion
Do not forget to run the action server if this is required by your bot, 
```

e.g. using:

.. code-block:: bash

  rasa run actions

> you need to supply a ``credentials.yml`` with the following content:

Reviewer: ```suggestion
You need to supply a ``credentials.yml`` with the following content:
```
